components/Navbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import Link from "next/link"
+import styles from '../styles/Nav.module.css'
+
+const Navbar = () => {
+  return (
+    <ul className={styles.ul}>
+      <li className={styles.li}>
+        <Link href="/">
+          <a>Home</a>
+        </Link>
+      </li>
+      <li className={styles.li}>
+        <Link href="/register">
+          <a>Register</a>
+        </Link>
+      </li>
+      <li className={styles.li}>
+        <Link href="/login">
+          <a>Login</a>
+        </Link>
+      </li>
+      {/* <Link href={user ? "/logout" : "/login"}>

```

Comment:
It's customary to remove commented code from a PR to main. Once it goes to main, always think that it'll go to production.

Reply:
Will keep that in mind sir. Thank you!